presto-oracle/src/test/java/io/prestosql/plugin/oracle/TestingOracleServer.java

```diff
@@ -40,7 +40,7 @@ public TestingOracleServer()
         // to fix this we have to change the number of processes of SPFILE
         // but this command needs a database restart and if we restart the docker the configuration is lost.
```

Comment:
> otherwise the tests end up giving a ORA-12519

this could mention the message too

> but this command needs a database restart and if we restart the docker the configuration is lost.

this could be solved with a custom entry point, or a command.
how much would it add to the startup time?

Reply:
@findepi The startup takes about 25 seconds. Do you mean https://www.testcontainers.org/features/commands/?

Reply:
Confirmed the parameters were set correctly.
```
SQL> SHOW PARAMETERS processes;

NAME				     TYPE	 VALUE
------------------------------------ ----------- ------------------------------
aq_tm_processes 		     integer	 0
db_writer_processes		     integer	 1
gcs_server_processes		     integer	 0
global_txn_processes		     integer	 1
job_queue_processes		     integer	 4
log_archive_max_processes	     integer	 4
processes			     integer	 1000
SQL>
SQL> SHOW PARAMETERS disk_asynch_io;

NAME				     TYPE	 VALUE
------------------------------------ ----------- ------------------------------
disk_asynch_io			     boolean	 FALSE
```

Reply:
bq. Do you mean https://www.testcontainers.org/features/commands/ and run /bin/bash /etc/init.d/oracle-xe restart?

This would work for tests, but would not be appropriate for product test environment (which we should have).